src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/FixedTermRecallsService.kt

```diff
@@ -47,60 +60,76 @@ class FixedTermRecallsService(private val featureToggles: FeatureToggles) {
     return postRecallReleaseDate != dates[CRD] && postRecallReleaseDate.isBefore(latestDateEntry.value)
   }
 
-  private fun getFixedTermRecallSentences(sentences: List<CalculableSentence>): List<CalculableSentence> {
-    return sentences.filter {
-      it.recallType == FIXED_TERM_RECALL_28 || it.recallType == FIXED_TERM_RECALL_14 &&
-        it.releaseDateTypes.contains(SLED)
-    }.sortedBy { it.sentencedAt }
+  private fun ftrIsMixedDuration(sentences: List<CalculableSentence>): Boolean {
```

Comment:
minor one - but you can get rid of some boiler plate code with doing this
private fun ftrIsMixedDuration(sentences: List<CalculableSentence>): Boolean =
    sentences.any {
      it.durationIsGreaterThanOrEqualTo(12, MONTHS)
    } && sentences.any {
      it.durationIsLessThan(12, MONTHS)
    }